sdk/identity/Azure.Identity/tests/WorkloadIdentityCredentialOptoinsTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Identity.Tests
+{
+    public class WorkloadIdentityCredentialOptoinsTests
+    {
+        [Test]
+        [NonParallelizable]
+        public void VerifyOptionsDefaultToEnvironmentValues([Values] bool specifyTenantId, [Values] bool specifyClientId, [Values] bool specifyTokenFilePath)
```

Comment:
validate null checks also?

Reply:
The options don't have any null checks. If these values are null the result will be a `CredentialUnavailableException` is thrown when tokens are requested. This is validated in `WorkloadIdentityCredentialTests.VerifyInvalidConfigurationThrowsCredentialUnavailable`